config/locales/en.yml

```diff
@@ -1091,6 +1091,7 @@ en:
             This field should be blank for instruments managed by reservation only.
             When blank or set to 0, the system will not cancel unstarted reservations.
           lock_window: "Window of time within which a customer may no longer move their reservation. Cancellations are permitted but may invoke a fee."
+          cutoff_time: "Minimum amount of hours before reservation instrument can be reserved."
```

Comment:
Maybe remove the "reservation". "Minimum number of hours before instrument can be reserved." NU has a lower priority ticket for this, but I think they might have a good idea of how to describe this.